scheduler/pmc-exporter-task/src/main/kotlin/uk/ac/ebi/scheculer/pmc/exporter/mapping/LinkMapper.kt

```diff
@@ -0,0 +1,18 @@
+package uk.ac.ebi.scheculer.pmc.exporter.mapping
+
+import ebi.ac.uk.extended.model.ExtSubmission
+import uk.ac.ebi.scheculer.pmc.exporter.model.Link
+import uk.ac.ebi.scheculer.pmc.exporter.model.Record
+import uk.ac.ebi.scheculer.pmc.exporter.model.Resource
+
+internal const val PMC_PATTERN = "S-E"
+internal const val BIOSTUDIES_URL = "http://www.ebi.ac.uk/biostudies/studies"
+
+object LinkMapper {
+    fun ExtSubmission.toLink(): Link {
```

Comment:
this does not have any state so I do not see reaon to declare inside an object but rather a plan extension